collector/metrics.go

```diff
@@ -74,7 +80,22 @@ func (e *Exporter) Collect(c chan<- prometheus.Metric) {
 		}
 	} else {
 		e.up.With(prometheus.Labels{"host": e.hostname}).Set(0)
-		log.Println("Could not fetch metrics from squid instance: ", err)
+		log.Println("Could not fetch counter metrics from squid instance: ", err)
 	}
+
+	insts, err = e.client.GetServiceTimes()
+
+	if err == nil {
+		e.up.With(prometheus.Labels{"host": e.hostname}).Set(1)
+		for i := range insts {
+			if d, ok := serviceTimes[insts[i].Key]; ok {
+				c <- prometheus.MustNewConstMetric(d, prometheus.CounterValue, insts[i].Value, e.labels.Values...)
+			}
+		}
+	} else {
+		e.up.With(prometheus.Labels{"host": e.hostname}).Set(0)
```

Comment:
Same as above. The error message is probably enough in this case too.

Reply:
Updated 👍